branding/java/code/src/com/redhat/rhn/branding/strings/StringResource_en_US.xml

```diff
@@ -2721,6 +2721,9 @@ You may grant provisioning entitlements _here_.
       <trans-unit id="States">
         <source>States</source>
       </trans-unit>
+      <trans-unit id="Recurring States">
+        <source>Recurring States</source>
+      </trans-unit>
```

Comment:
As @mcalmer mentioned recently we should no longer add strings to `/branding`. IIUC we are currently moving all the strings in branding back to the original location as we are the upstream now.